### PR TITLE
Added missing readme files for components

### PR DESCRIPTION
**Pull request by Rubemlrm**
Added readme.md files for the following componenents:

- cache
- container
- database
- encryption
- events
- logs
- middleware
- pagination
- routing
- session
- support
- translation
- validation
- view
- other-components/html

I had some doubts relative to the documentation of support component, since i can't find a page on documentation relative to that, i have linked to some useful information pages based on the examples.

### CODE REVIEW

File: components/cache/readme.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# <img src="../../torch-logo.png" alt="Torch Logo" align="right">Torch - Using Laravel's Illuminate Components Independently
+
+[&lt;&lt; Back to Torch](../../readme.md)
+
+# Config
```
Comment (mattstauffer) [inline components/cache/readme.md]: You left the page title the same for each; you'll want this to say "Cache" on this one, etc.

File: components/support/readme.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# <img src="../../torch-logo.png" alt="Torch Logo" align="right">Torch - Using Laravel's Illuminate Components Independently
+
+[&lt;&lt; Back to Torch](../../readme.md)
+
+# Config
+
+This component shows how to use Laravel's **Support** features in non-Laravel applications.
+
+Since this component dont have a dedicated page on doc, you can use the following links for more information:
```
Comment (m1guelpf) [inline components/support/readme.md]: doesn't instead of dont

File: components/cache/readme.md (hunk 1 above)
Comment (Rubemlrm) [reply components/cache/readme.md]: Fixed the page titles , sorry was distracted .

**Comment by mattstauffer** [comment]
@Rubemlrm Thanks! This is a big improvement!

**Comment by Rubemlrm** [comment]
that was a pleasure to help =) i will keep an eye on the repo in case more component are added and need something else .

**Pull request merged**